publication-commons/src/main/java/no/unit/nva/publication/model/storage/DynamoEntryByIdentifier.java

```diff
@@ -17,54 +16,54 @@
 import nva.commons.core.SingletonCollector;
 
 public interface DynamoEntryByIdentifier {
-    
+
     @JsonProperty(BY_TYPE_AND_IDENTIFIER_INDEX_PARTITION_KEY_NAME)
     default String getByTypeAndIdentifierPartitionKey() {
         return entryTypeAndIdentifier();
     }
-    
+
     @JsonProperty(BY_TYPE_AND_IDENTIFIER_INDEX_SORT_KEY_NAME)
     default String getByTypeAndIdentifierSortKey() {
         return entryTypeAndIdentifier();
     }
-    
+
     SortableIdentifier getIdentifier();
-    
+
     Entity getData();
-    
-    default Dao fetchByIdentifier(AmazonDynamoDB client) throws NotFoundException {
+
+    default Dao fetchByIdentifier(AmazonDynamoDB client, String tableName) throws NotFoundException {
```

Comment:
I prefer if we pass the table name in the constructor if that is possible. It is only the import-candidates lambdas that operates with several tables, and I think it's okay if you have two resourceServices you operate with. Like you would if you are reading and writing to two different s3 buckets you would need two s3Drivers.

Reply:
This ones is passed through resource service constructor, but this method is used by daos.